gnd/src/main/java/com/google/android/gnd/ui/editrecord/EditRecordFragment.java

```diff
@@ -207,7 +207,7 @@ public void addMultipleChoiceField(Field field) {
 
   public void onShowDialog(Field field) {
     Cardinality cardinality = field.getMultipleChoice().getCardinality();
-    Optional<Value> currentValue = viewModel.getValue(field.getId());
+    Optional<Response> currentValue = viewModel.getValue(field.getId());
```

Comment:
Is the response/value distinction something we're strictly adhering to at the type level only? To me, it's easier if I can think of "answer to form field" as a single thing called a response in general (at both the type and term level).

In other words, this would be called `currentResponse`

Reply:
Right, Response is only defined in the context of Records, as the user's response to a particular question.